tests/test_bug_features.py

```diff
@@ -165,6 +166,14 @@ def test_comment_length(read):
     read("comment_length.json", comment_length, [566, 5291])
 
 
+def test_delta_nightly_request_merge(read):
+    read(
+        "nightly_uplift.json",
+        delta_nightly_request_merge,
+        [pytest.approx(6.4318), pytest.approx(0.87326389)],
```

Comment:
In general it is not a good practice to compare float values over "==", as python internal representation of fractions may be different. I have changed to precise values but still wrapped it in pytest.approx() in case client/developer use different python version or different platform/hardware.